charts/gke.md

```diff
@@ -0,0 +1,87 @@
+# Install Zenko On Google Cloud Kubernetes Engine
+
+Zenko can be installed on a cluster managed on Google Cloud (GKE).
+Start a new cluster on Kubernetes following the instructions on [Google
+Cloud
+documentation](https://cloud.google.com/kubernetes-engine/docs/quickstart).
+
+To run Zenko you'll need a cluster with 3 nodes, 2vCPU and 7.5GB RAM
+each. Once the cluster is running, connect to it and install Helm on
+it.
+
+## Create Role For Tiller
+
+Google Kubernetes Engine requires Role Based Access Control to be
+setup. The first step is to create a `serviceaccount` for `tiller`:
+
+```shell
+$ kubectl create serviceaccount tiller --namespace kube-system
+```
+
+## Install Helm on GKE Cluster
+
+Strangely, Helm is not available by default on GKE and needs to be
+installed.
+
+```shell
+$ curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get > get_helm.sh
+$ bash ./get_helm.sh
+```
+
+Once that's completed, start Helm:
+
+```shell
+$ helm init --service-account tiller --wait
+```
+## Install Zenko
+
+Clone Zenko's repo and go into the charts directory:
+
+```shell
+$ git clone https://github.com/scality/Zenko.git
+$ cd ./Zenko/charts
+```
+
+Once you have the repo cloned you can retrieve all dependencies:
+
+```shell
+$ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+"incubator" has been added to your repositories
+
+$ helm dependency build zenko/
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "incubator" chart repository
+...Successfully got an update from the "stable" chart repository
+Update Complete. ⎈Happy Helming!⎈
+Saving 8 charts
+Downloading prometheus from repo https://kubernetes-charts.storage.googleapis.com/
+Downloading mongodb-replicaset from repo https://kubernetes-charts.storage.googleapis.com/
+Downloading redis from repo https://kubernetes-charts.storage.googleapis.com/
+Downloading kafka from repo http://storage.googleapis.com/kubernetes-charts-incubator
+Downloading zookeeper from repo http://storage.googleapis.com/kubernetes-charts-incubator
+Deleting outdated charts
+```
+
+With your dependencies built, you can run the following shell command
+to deploy a single node Zenko stack with Orbit enabled.
+
+
+```shell
+$ helm install --name zenko zenko
```

Comment:
Typically you may want to enable the ingress on Zenko like this:
```helm install --name zenko --set ingress.enabled=true zenko```. Which does require a running ingress controller. I think GKE runs an GCE ingress controller by default but I can't find an official reference to it yet.

Reply:
Better be explicit then. I amended the commit.